hibernate-reactive-core/build.gradle

```diff
@@ -71,7 +71,7 @@ tasks.withType(Test) {
     defaultCharacterEncoding = "UTF-8"
     testLogging {
         displayGranularity 1
-        showStandardStreams = false
+        showStandardStreams = project.hasProperty('showStandardOutput') ? getProperty('showStandardOutput').toBoolean() : false
```

Comment:
On second thought, I think you can change this to
```
showStandardStreams = project.hasProperty('showStandardOutput')` 
```

We can assume that if somebody pass the property is because they want to enable it. Not reason to check the actual value.

Reply:
After that I think you can merge it

Reply:
agreed

pushed those changes. Will merge when build complete, thx